src/flowr/internal-session.ts

```diff
@@ -93,15 +90,21 @@ export class FlowrInternalSession {
 			if(blockedLines.has(i)) {
 				continue
 			}
-			diagnostics.push({
-				message:  `irrelevant when slicing for '${token}' (line: ${pos.line + 1}, char: ${pos.character + 1})`,
+			ret.push({
+				message:  `irrelevant when slicing for '${document.getText(range)}' (line: ${pos.line + 1}, char: ${pos.character + 1})`,
 				range:    new vscode.Range(i, 0, i, document.lineAt(i).text.length),
 				severity: vscode.DiagnosticSeverity.Hint,
 				tags:     [vscode.DiagnosticTag.Unnecessary]
 			})
 		}
-		this.diagnostics.set(uri, diagnostics)
-		this.outputChannel.appendLine('slice: ' + JSON.stringify([...result.slice.result]))
-		return result.reconstruct.code
+		return ret
+	}
+
+	public static fixEncoding(text: string) {
```

Comment:
this is definitely hacky and - i think - no longer necessary after we dropped xmlparsedata!

Reply:
#32